focus-group.js

```diff
@@ -109,3 +107,36 @@ export function menuKeyUX(options) {
     }
   }
 }
+
+export function findGroupNodeByEventTarget(eventTarget) {
```

Comment:
Do we need to export them?

Removing export will reduce JS bundle size

Reply:
I use export for testing purposes only, probably there is a better way to test the method...

Reply:
Can we do integration testing instead (test the whole feature)?

Reply:
Sure I just don't know which strategy is better here... I will try to fix it today evening!

Reply:
done